controller/proxy-injector/webhook_test.go

```diff
@@ -61,26 +62,58 @@ func TestMutate(t *testing.T) {
 	}
 }
 
-func TestIgnore(t *testing.T) {
-	ns, err := factory.Namespace("namespace-kube-public.yaml")
+func TestShouldInject(t *testing.T) {
+	nsEnabled, err := factory.Namespace("namespace-kube-public.yaml")
```

Comment:
naming nit: mind calling these files `namespace-inject-enabled.yaml` and `namespace-inject-disabled.yaml`